mantidproject/mantid
# QENS fit function: isotropic rotational diffusion

The QENS fit function for an isotropic rotator:  
Parameters:  r: rotator radius;  tau: relaxation time  
`S(Q,E) = A_0(Qr) * \delta(Q) + sum_{l-1}^N A_l(Qr) * 1/pi * HWHM_l / (HWHM_l^2 + E^2)`
with 
`A_0(Qr) = j_0(Qr)^2    # j_0: spherical bessel function`
`A_l(Qr) = (2l+1)j_l(Qr)^2`
`HWHM_l = l*(l+1)*\hbar / tau`

**To test:**
Donwload and unzip file 
[testIsoRotDiff.zip](https://github.com/mantidproject/mantid/files/520496/testIsoRotDiff.zip) containing:
- `data.nxs`  : QENS signal
- `resolution.nxs`: resolution function
- `globalFit.py`: script to do a fit of all spectra of `data.nxs` with `IsoRotDiff`

Load the QENS signal and the resolution. Then load the fit script in the python window and execute. The Fit should obtain optimized rotation radius R=0.45 Angstroms and relaxation time Tau=26 ps.  

Alternatively, do a manual fit of a single spectrum. For this, load the following model in the fit window:
`(composite=Convolution,FixResolution=true,NumDeriv=true;name=TabulatedFunction,Workspace=resolution,WorkspaceIndex=7,Scaling=1,Shift=0,XScaling=1;(name=IsoRotDiff,N=25,NumDeriv=false,Q=1.9,f0.Height=1.09435,f0.Centre=9.8649e-08,f0.Radius=0.48702,Tau=10))`.
This model is the initial guess for the fit of the last spectrum in `data.nxs` (Q=1.9).

Fixes #17555

[Release notes](https://github.com/mantidproject/mantid/blob/a6fc9a6824cb66fee364165844481a1fe8d8d15a/docs/source/release/v3.9.0/indirect_inelastic.rst) have been updated.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
